src/site/site.xml

```diff
@@ -21,74 +21,49 @@
 
 <project xmlns="http://maven.apache.org/DECORATION/1.0.0"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://maven.apache.org/DECORATION/1.0.0 http://maven.apache.org/xsd/decoration-1.0.0.xsd">
+    xsi:schemaLocation="http://maven.apache.org/DECORATION/1.0.0 http://maven.apache.org/xsd/decoration-1.0.0.xsd"
+    name="Apache HBase">
+  <publishDate position="none"/>
+  <version position="none"/>
+
   <skin>
-    <!-- We tried to get some changes to the fluido-skin committed upstream
-         but with little success. This is a custom artifact that contains
-         some commits over the official maven-fluido-skin 1.4 release. The
-         actual code is available at:
-         https://github.com/joshelser/maven-fluido-skin/tree/1.4-HBase-patched.
-         The hope is that we can get these fixes included upstream and
-         remove the need for this custom fork. This is published to
-         my end-user's GAV for ease of releasing this to maven central
-         until the upstream update happens and is released.
-         See HBASE-14785 and HBASE-21005 for more info. -->
-    <groupId>org.apache.maven.skins</groupId>
-    <artifactId>maven-fluido-skin</artifactId>
-    <version>1.12.0</version>
+    <groupId>io.github.devacfr.maven.skins</groupId>
+    <artifactId>reflow-maven-skin</artifactId>
+    <version>${reflow-maven-skin.version}</version>
   </skin>
+
   <custom>
-    <fluidoSkin>
-      <topBarEnabled>true</topBarEnabled>
-      <sideBarEnabled>false</sideBarEnabled>
-      <googleSearch>
-        <!-- The ID of the Google custom search engine to use.
-             This one searches hbase.apache.org, issues.apache.org/browse/HBASE-*,
-             and user and dev mailing list archives. -->
-        <customSearch>000385458301414556862:sq1bb0xugjg</customSearch>
-      </googleSearch>
```

Comment:
I did not found a solution (yet) to use this Google custom search engine with the newer skin.
Do you think it is still needed? Because most probably this can be solved somehow but may require more work / hacks.

Reply:
Huh, I didn't know that it had the behavior described in the comment.

Is it this? https://programmablesearchengine.google.com/about/

Reply:
Yeah it might be. But I think we don't use this, just the Google site search: search redirects to e.g.

https://www.google.com/search?sitesearch=hbase.apache.org&q=canary

when we search for "canary".